packages/zosfiles/__tests__/__unit__/methods/copy/Copy.unit.test.ts

```diff
@@ -84,7 +84,7 @@ describe("Copy", () => {
                     expect(copyExpectStringSpy).toHaveBeenLastCalledWith(
                         dummySession,
                         expectedEndpoint,
-                        expectedHeaders,
+                        expect.arrayContaining(expectedHeaders),
```

Comment:
We want to make sure that no extra headers are being defined, so I don't think we should use `expect.arrayContaining` for headers in our unit tests.

If we need a way to ignore the order of the headers, we could create a Jest helper function that expects the exact same set of items but allows any order.

Reply:
Instead of `expect.arrayContaining`, we may able to use the `expect. toIncludeSameMembers` helper function provided by the [jest-extended](https://www.npmjs.com/package/jest-extended) package.

Reply:
ive implemented the jest-extended package but i think i still need to keep the expect.arrayContaining because the first expect statment that includes arrayContaining is about making sure the spy was called with the right arguments. the next expect is more granular about the headers.